pages/register.js

```diff
@@ -47,10 +60,87 @@ const RegistrationForm = () => {
                         justifyItems="start"
                         overflowY="auto"
                     >
-                        <FieldSection title='First Name' />
-                        <SystemComponent>as</SystemComponent>
-                        <SystemComponent>as</SystemComponent>
+                        <FieldSection 
+                        title='Full Name '
+                        asterik={true}                     
+                            name="fullName"
+                            value={formValues['fullName']}
+                            onChange={handleInputChange}        
+                            />
+                        <SystemComponent></SystemComponent>
+                    </SystemComponent>
+                    <SystemComponent
+                        display="grid"
+                        gridTemplateColumns="1fr"
+                        gridAutoRows="autofill"
+                        gridAutoFlow="row"
+                        gridGap={["cardPadding", "cardMargin", "cardMargin"]}
+                        justifyItems="start"
+                        overflowY="auto"
```

Comment:
No, don't repeat this SystemComponent with the property `display: grid` there should only be one grid on this page, that holds all the form inputs/textboxes.